### ISSUE TITLE
NTR: Mesaconic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8639]
New term request Mesaconic acid high in urine (elthat_190101150426): Citraconic acid; 2-methyl-2-butenedioic acid; methylfumaric acid; monomethylfumarate

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000465
- label: Elevated urine mesaconic acid level
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: PMID:29909962
- def: The amount of mesaconic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8639
- synonyms:
